### ISSUE TITLE
Support "delayed expansion"

### ISSUE THREAD

**Issue by BooDoo** [BooDoo/CheapBotsTootSweet#16]
An [example provided by a user](https://twitter.com/SerinDelaunay/status/1037389841707745282), which works in Tracery1 but fails in Tracery2:
```json
{ 
  "origin": "#[tweet:#start#]tweet#", 
  "start": "\\#helloworld\\#", 
  "helloworld": "omg what how"
}
```
It would be cool to find a way to carry this over into full Tracery2 migration (#13)
Either by fiddling with the escaping (#10) or, perhaps better, by way of a Tracery "modifier"?

**Comment by BooDoo** [comment]
wait why does this magically work now, 4 years later.

Was this just a bad example case? Am I misremembering what the intended output is.

**Comment by BooDoo** [comment]
Resolved by 9a74f26 / BooDoo/traceryhosting-frontend@acb4cc7

It is no longer necessary (**or supported**) to double-slash escape delayed expansion tokens as in the original Issue write-up. This is now written simply, and intuitively, as 
```json
{ 
  "origin": "#[tweet:#start#]tweet#", 
  "start": "#helloworld#", 
  "helloworld": "omg what how"
}
```
